WordPress/twentytwentyfive
# Pattern: Intro: Short heading and paragraph and image on the right

**Description**
Adds the Short heading and paragraph and image on the right side.

**Screenshots**
<img width="1150" alt="Screenshot at Aug 27 12-15-15" src="https://github.com/user-attachments/assets/d836def7-8684-4057-bc41-c4ed8ab06747">


**Testing Instructions**

1. Add the pattern and check if it matches the design.
2. Also change the width of the browser window and check that it still looks correct.
